projects/kit/components/multi-select/native-multi-select/native-select-group.component.ts

```diff
@@ -0,0 +1,42 @@
+import {ChangeDetectionStrategy, Component, Input, TemplateRef} from '@angular/core';
+import {TuiDataListDirective} from '@taiga-ui/core';
+
+import {AbstractTuiNativeSelect} from './native-select';
+
+@Component({
+    selector: 'select[multiple][tuiSelect][labels]',
+    templateUrl: './native-select-group.template.html',
+    providers: [
+        {
+            provide: TuiDataListDirective,
+            deps: [TuiNativeSelectGroupComponent],
```

Comment:
How can `useExisting` have `deps`? 🤔  If it's a mistake, maybe it's time for `tuiAsDataList` helper?